main.go

```diff
@@ -109,8 +109,17 @@ func scrapeStreams(twitch *Client) {
 		for id := range channelsID {
 			streamsID = append(streamsID, id)
 		}
-
 		var streamScraped = 0
+
+		// if access token expire in 10 days renew it
+		if time.Now().After(twitch.Token.ExpiresDate.AddDate(0, 0, -10)) {
```

Comment:
Why not using a percent remaining like 50% ? If twitch reduce to 24H the code will crash